bundle-core/src/main/java/net/discdd/bundlesecurity/ServerSecurity.java

```diff
@@ -377,7 +385,13 @@ public long getCounterFromBundlePath(Path bundlePath, boolean direction) throws
         byte[] encryptedBundleID = Files.readAllBytes(bundleIDPath);
         String receivedBundleID, latestBundleID;
 
-        byte[] clientIdentityKeyBytes = SecurityUtils.decodePublicKeyfromFile(bundlePath.resolve(CLIENT_IDENTITY_KEY));
+        //create server secuirty instance
```

Comment:
there is a misspelling. i don't think these two comments are informative. remove?